pom.xml

```diff
@@ -728,7 +728,26 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-site-plugin</artifactId>
-          <version>3.4</version>
+          <version>3.3</version>
```

Comment:
Does Oryx use a 2 space indent and a 4 space overhang? I'll change the settings in my editor.